Dockerfile

```diff
@@ -23,7 +23,7 @@ RUN pnpm install --offline
 ######################################################################
 # Stage name "development" is required for development with dargstack.
 # TODO: try if slim or alpine versions work.
-FROM python:3.11.4-buster@sha256:3a19b4d6ce4402d11bb19aa11416e4a262a60a57707a5cda5787a81285df2666 AS development
```

Comment:
cc @soerface @malled2002 this updates the debian base image from buster to bookworm, just fyi in case there is an issue with deployment

Reply:
ah well nvm, this isn't deployed through docker, docker just runs the CI

Reply:
@dargmuesli Doesn't boot anymore. On the pi, we have python 3.11 installed. The project now requires 3.13.

I'll roll back the project requirement. Desired solution in my opinion: Use uv, so that the correct python version is automatically installed.

Reply:
Yes, do the UV!

Reply:
Also keep in mind that the pre-commit tool's Python version may need to match in CI